citi-onboarding/Aqua-Jr
# feature/aboutUs-testimony-dataBase-routes

## What I did
- Created the db for testimony and about us section
- Created the api route for those dbs

## How to test
1. Open the project in terminal
2. cd server/
3. nodemon index.js
5. In your browser, go to http://localhost:3001/admin/about-us and add a item
6. go to http://localhost:3001/api/aboutus and check if the item you added was really added
7. In your browser, go to http://localhost:3001/admin/testimonies and add a item
8. go to http://localhost:3001/api/testimony and check if the item you added was really added
